client/components/forms/form-country-select/index.jsx

```diff
@@ -34,7 +34,7 @@ export default localize( React.createClass( {
 
 		return (
 			<select
-				{ ...omit( this.props, 'className' ) }
+				{ ...omit( this.props, [ 'className', 'countriesList', 'translate', 'moment', 'numberFormat' ] ) }
```

Comment:
I really thought `htmlFor` was going to swing @klimeryk, but the only other relevant attribute we use much at the moment is `autoFocus`, and that's a lot of good devs going the other way, so I'll switch to the whitelist approach.

We can always change again later if we start making more use things like [aria attributes](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/forms/Basic_form_hints#Required_and_invalid_fields).